devel/posix-manpages/Portfile

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                posix-manpages
+version             2013
+categories          devel
+platforms           darwin
+supported_archs     noarch
+license             Other
```

Comment:
That's not a helpful value for the `license`. It won't be distributable. See https://trac.macports.org/wiki/PortfileRecipes#license for assistance.

Reply:
I've changed it to `Permissive` based on the discussion [here](https://lwn.net/Articles/581858/).